drivers/network/dd/nvnet/netnv.inf

```diff
@@ -142,14 +142,12 @@ HKR, Ndi\params\OptimizationMode,      default,   0, "0"
 HKR, Ndi\params\OptimizationMode\enum, "0",       0, "%M1%"
 HKR, Ndi\params\OptimizationMode\enum, "1",       0, "%M2%"
 HKR, Ndi\params\OptimizationMode\enum, "2",       0, "%M3%"
-
 HKR, Ndi\params\NetworkAddress,        ParamDesc, 0, %NA%
 HKR, Ndi\params\NetworkAddress,        type,      0, "edit"
 HKR, Ndi\params\NetworkAddress,        LimitText, 0, "12"
 HKR, Ndi\params\NetworkAddress,        UpperCase, 0, "1"
 HKR, Ndi\params\NetworkAddress,        default,   0, " "
 HKR, Ndi\params\NetworkAddress,        optional,  0, "1"
-
 HKR, Ndi\params\SpeedDuplex,           ParamDesc, 0, %SD%
 HKR, Ndi\params\SpeedDuplex,           type,      0, "enum"
 HKR, Ndi\params\SpeedDuplex,           default,   0, "0"
```

Comment:
These removed empty lines served a purpose to separate different driver parameters and make them more readable and distinguishable between each other. Now it's all mixed up.

Reply:
No it is very easy to differentiate both, because the paths do distinguish. Open the modified file within an editor, and see for yourself.

Reply:
But if you insist on it, I can change this part back, because it is at least not within the strings section, where the newline already has a different meaning.

Reply:
Changed back.